packages/tsconfig/react-library.json

```diff
@@ -10,6 +10,7 @@
     "module": "ESNext",
     "rootDir": "src",
     "outDir": "dist",
-    "jsx": "react"
+    "jsx": "react",
+    "downlevelIteration": true
```

Comment:
Måtte legge til denne for å kunne spreade collections som implementerer Iterable-interfacet. Burde ikke ha noen særlig påvirkning på størrelsen av bundlen.